ui/app/components/auth-form.js

```diff
@@ -11,57 +12,126 @@ const DEFAULTS = {
 
 export default Ember.Component.extend(DEFAULTS, {
   classNames: ['auth-form'],
-  routing: inject.service('-routing'),
+  router: inject.service(),
   auth: inject.service(),
   flashMessages: inject.service(),
+  store: inject.service(),
   csp: inject.service('csp-event'),
+
+  // set during init and potentially passed in via a query param
+  selectedAuth: null,
+  methods: null,
+  cluster: null,
+  redirectTo: null,
+
   didRender() {
+    this._super(...arguments);
     // on very narrow viewports the active tab may be overflowed, so we scroll it into view here
-    this.$('li.is-active').get(0).scrollIntoView();
+    let activeEle = this.element.querySelector('li.is-active');
+    if (activeEle) {
+      activeEle.scrollIntoView();
+    }
+    activeEle = null;
```

Comment:
Little nit for the minute, do you need this nulling here?

Reply:
Heh whoops, been a while since doing DOM-y things and was just overzealous. I imagine it doesn't hurt, but also probably not needed.